telegram/constants.py

```diff
@@ -52,3 +52,42 @@
 MAX_MESSAGES_PER_MINUTE_PER_GROUP = 20
 MAX_MESSAGE_ENTITIES = 100
 MAX_INLINE_QUERY_RESULTS = 50
+
+
+class MediaCategory:
```

Comment:
Just put these constants in the filters.py, since that's the only place they are used and they aren't meant to be exported to the general user.

Reply:
Not necessary if I use built-in mimetypes